alacritty/src/renderer/text/glyph_cache.rs

```diff
@@ -104,6 +98,22 @@ impl GlyphCache {
         })
     }
 
+    // Get metrics adjusted for glyph offset.
+    fn get_offset_adjusted_metrics(
```

Comment:
```suggestion
    fn load_font_metrics(
```

And a CHANGELOG entry.